src/main/webapp/timestamps.js

```diff
@@ -39,3 +39,13 @@ function timestampToString(timestampMs) {
 function secondsToMilliseconds(seconds) {
   return Math.round(seconds * MILLISECONDS_PER_SECOND);
 }
+
+/**
+ * @returns {string} A time range from
```

Comment:
You do not need to use @return if you only have 1 line of comment. Please write it as:

Returns a time range string in the forma of "`startTimestampMs` - `endTimestampMs`"

Or something like that.

Reply:
+1

Reply:
done, thanks!